angular-ngrx-scss/src/app/shared/components/issue-pull-item/issue-pull-item.component.scss

```diff
@@ -0,0 +1,65 @@
+@use '../../styles/variables.scss';
+
+.item {
+  padding: 1.125rem;
+  border-bottom: 1px solid variables.$gray200;
+
+  &:hover {
+    background-color: variables.$gray100;
+  }
+}
+
+.item-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .info {
+    display: flex;
+  }
+
+  .comments {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.icon {
+  margin-right: 0.5rem;
+
+  &.open {
+    color: variables.$green100
```

Comment:
Not sure what the conventions are, but a small suggestion: I would create local variables for the "open" and "closed" colors, so we can easily change them in this stylesheet later if needed.
Something like
`$color-open: variables.$green100;
$color-closed: variables.$purple100;`
on top of this file.